bricksrc/sensor.py

```diff
@@ -1666,6 +1666,63 @@
                             },
                         },
                     },
+                    "Radiant_Panel_Temperature_Sensor": {
+                        "tags": [
+                            TAG.Point,
+                            TAG.Sensor,
+                            TAG.Temperature,
+                            TAG.Radiant,
+                            TAG.Panel,
+                        ],
+                        "substances": [
+                            [BRICK.measures, BRICK.Temperature],
+                            [BRICK.measures, BRICK.Radiant_Panel],
+                        ],
+                        "subclasses": {
+                            "Inside_Face_Surface_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Inside,
+                                    TAG.Face,
+                                    TAG.Surface,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                            },
+                            "Outside_Face_Surface_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Outside,
+                                    TAG.Face,
+                                    TAG.Surface,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                            },
+                            "Embedded_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Embedded,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                                "substances": [
+                                    [BRICK.measures, BRICK.Temperature],
+                                    [BRICK.measures, BRICK.Internal_Material],
```

Comment:
Let's leave off the `[BRICK.measures, BRICK.Internal_Material]` for now. I think the better model would be to have `Concrete_Temperature_Sensor`, etc variations for the different kinds of materials that might exist in a radiant panel? Then we have the question if we need to capture whether it is Embedded Temperature or Core Temperature: is this a crucial distinction, and if so, why is Core a subclass of Embedded rather than being a sibling class?

Reply:
I agree that we could leave off the generic "internal material" to replace it for the specific materials.

The idea with the class `Embedded_Temperature_Sensor` is to be the generic version of a temperature measurement inside a concrete slab. We often don't know how deep the sensor is in the slab. The idea with `Core_Temperature_Sensor` was to specify that the measurement is at the same level as the hydronic pipe of the radiant system. The better way might be just to have one of the classes and then add the sensor depth by adding a property to the class.

Reply:
Guess it broke when that property got removed. Let me give some thought for how to model this more appropriately